recipes/tmhg/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tmhg" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: 'https://github.com/yongze-yin/tMHG-Finder/archive/refs/tags/v{{version}}.tar.gz'
+  sha256: a05df84ca80674154cb78f63148e26f023b88af26fa95cc2fa54e34f105efcac
+
+build:
+  run_exports:
+    - {{ pin_subpackage('tmhg', max_pin="x") }}
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  host:
+    - python >=3.6,<3.9
```

Comment:
those are quite old python version, is there a plan to support newer versions as well?